### PR TITLE
Insight Docs Tweaks

### PR DESCRIPTION
**Pull request by nataliepina**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/insights/insights-view/index.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ enterprise: false
 techPreview: true
 ---
 
-<p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis. Older insights drop off the table, but, if an insight persists, a new insight item populates on the board when it refreshes.</p>
+<p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis. Older insights expire, however, if an insight persists, it reappears in the table.</p>
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/insights/insights-view/index.md:12]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis. Older insights expire. However, if an insight persists, it reappears in the table.</p>
```

File: pages/dkp/kommander/2.2/insights/insights-view/index.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ enterprise: false
...
 
-After the [Insights Engine is enabled](../insights-setup/), select one of your workspaces to view your insight items. From a Workspace Dashboard, the Insights summary table displays the most recent insight items, below the workspace summary cards. You can view Critical, Warning, Notices, and View All for a summary of insight items.
+After the [Insights Engine is enabled](../insights-setup/), select a workspaces to view your insight items. On the Workspace Dashboard below the workspace summary cards, the Insights summary card displays the most recent insight items. The Insights summary card displays the amount of insights within each severity level of Critical, Warning, and Notices. Select the View All link to see a summary of all insight items on the Insights Dashboard.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/insights/insights-view/index.md:14]:
```suggestion
After the [Insights Engine is enabled](../insights-setup/), select a workspace to view your insight items. On the Workspace Dashboard below the workspace summary cards, the Insights summary card displays the most recent insight items. The Insights summary card displays the amount of insights within each severity level of Critical, Warning and Notices. Select the View All link to see a summary of all insight items on the Insights Dashboard.
```

File: pages/dkp/kommander/2.2/insights/insights-view/index.md (hunk 1 above)
Comment (cmastr) [inline pages/dkp/kommander/2.2/insights/insights-view/index.md:12]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Insights are refreshed every 72-hours. Older insights expire, but if an insight persists, it reappears in the table.</p>
```

**Review by mmcelroy91 — APPROVED** [review]

**Review by kaiwalyajoshi — APPROVED** [review]

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
     EOF
     ```
 
-    This grants all Attached clusters the ability to enable the Insights Engine, and DKP Insights icon appears under Applications.
+    This grants all Attached clusters the ability to enable the Insights Engine, and displays DKP Insights under Applications.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:39]: I'm just not sure what the correct subject verb agreement is here - so making it to display

```suggestion
    This grants all Attached clusters the ability to enable the Insights Engine, and to display DKP Insights under Applications.
```

File: pages/dkp/kommander/2.2/insights/insights-view/index.md [hunk 4]
```diff
@@ -9,11 +9,11 @@ enterprise: false
...
 
-Select Insights from the sidebar menu for the DKP Insights Dashboard, where you can filter the insight items several different ways:
+Select **Insights** from the sidebar menu for the DKP Insights Dashboard, where you can filter the insight items several different ways:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/insights/insights-view/index.md:16]: I think this is the standard that we call it?

```suggestion
Select **Insights** from the side nav menu for the DKP Insights Dashboard, where you can filter the insight items several different ways:
```

**Review by Colin1841 — APPROVED** [review]

**Review by msdolbey — APPROVED** [review]
👍 
looks good to me

**Comment by nataliepina** [comment]
Combined all the feedback and updated the PR. Thanks, y'all! 🎉

**Pull request merged**